Lib/multiprocessing/queues.py

```diff
@@ -36,6 +38,9 @@ def __init__(self, maxsize=0, *, ctx):
         if maxsize <= 0:
             # Can raise ImportError (see issues #3770 and #23400)
             from .synchronize import SEM_VALUE_MAX as maxsize
+        elif maxsize > SEM_VALUE_MAX:
+            maxsize = SEM_VALUE_MAX
+        warnings.warn(f"maxsize exceeds SEM_VALUE_MAX; setting maxsize to {SEM_VALUE_MAX}", UserWarning)
```

Comment:
```suggestion
            warnings.warn(f"maxsize exceeds SEM_VALUE_MAX; setting maxsize to {SEM_VALUE_MAX}", UserWarning)
```

Reply:
It looks like the indent may have been off

Reply:
Yes you are right the indent was off, thank you.